data/campaigns/The_Rise_Of_Wesnoth/scenarios/06_Temple_in_the_Deep.cfg

```diff
@@ -229,7 +229,7 @@
 
                         [message]
                             speaker=narrator
-                            message= _ "As you open the chest you see it, the Ruby of Fire. It is the size of an apple, and burns with an internal fire, which is refracted through its faces. You can feel the power flowing from it..."
+                            message= _ "As Haldric opens the chest he sees it, the Ruby of Fire. It is the size of an apple, and burns with an internal fire, which is refracted through its faces. He can feel the power flowing from it..."
```

Comment:
I think I have a few punctuation complaints here too.

* Shouldn't there be a comma after "chest"?
* The comma before "Ruby of Fire" doesn't seem right. I think it should be a dash or maybe even a colon. (And I don't suggest colons lightly; they're usually the wrong choice.)
* I'm not sure about the comma after "apple". It seems to me that it could be omitted, maybe?

Reply:
>Shouldn't there be a comma after "chest"?

Sounds good to me since the introductory clause is long.

> The comma before "Ruby of Fire" doesn't seem right. I think it should be a dash or maybe even a colon. (And I don't suggest colons lightly; they're usually the wrong choice.)

I'll go with an em dash for this one.

> I'm not sure about the comma after "apple". It seems to me that it could be omitted, maybe?
We can omit. The clause after apple is a dependent clause after all.